### PR TITLE
Allow sections be not commentable when the hearing isn't

### LINKED ISSUE #216: Let single sections not be commentable even if the hearing is commentable
Currently, hearingAllowsComments for the whole hearing is passed directly onto SectionList and all Sections.

Therefore, all sections look like commentable on the UI even if they are not commentable in the API. Trying to post a Comment on such a section results in an API error.

### PR DESCRIPTION
**Pull request by tuomas777**
Closes #216

### CODE REVIEW

File: src/components/SectionList.js [hunk 1]
```diff
@@ -41,6 +42,7 @@ SectionList.propTypes = {
   sections: React.PropTypes.array,
   sectionComments: React.PropTypes.object,
   nComments: React.PropTypes.number,
+  user: React.PropTypes.object
```
Comment (akx) [inline src/components/SectionList.js]: Add a comma.

**Comment by tuomas777** [comment]
added a comma

File: src/components/Section.js [hunk 2]
```diff
@@ -168,4 +168,5 @@ Section.propTypes = {
   onPostVote: React.PropTypes.func,
   loadSectionComments: React.PropTypes.func,
   comments: React.PropTypes.object,
+  user: React.PropTypes.object
```
Comment (akx) [inline src/components/Section.js]: This could use that trailing comma too. :D

**Comment by tuomas777** [comment]
added a second comma

**Pull request merged**